src/config/dark-sites.config

```diff
@@ -686,6 +690,7 @@ polsatboxgo.pl
 polsatgo.pl
 pony.tube
 pool.pm
+pornhub.com
```

Comment:
NSFW Sites aren't permitted.

Reply:
Is that a rule? I've checked [CONTRIBUTING.md](https://github.com/darkreader/darkreader/blob/7daacbeddd59e7742c21bca8a5fb371cd72962e7/CONTRIBUTING.md) and searched through the project's files and it doesn't seem to be listed anywhere. Can you add that to CONTRIBUTING.md?

Reply:
It's more or less a unwritten rule at the moment.